src/main/java/nl/haris/backendspringboottechiteasymodel/controllers/TelevisionController.java

```diff
@@ -1,39 +1,46 @@
-package nl.haris.backendspringboottechiteasycontroller.controllers;
+package nl.haris.backendspringboottechiteasymodel.controllers;
 
-import nl.haris.backendspringboottechiteasycontroller.Television;
-import nl.haris.backendspringboottechiteasycontroller.exceptions.NameTooLongException;
-import nl.haris.backendspringboottechiteasycontroller.exceptions.RecordNotFoundException;
+import nl.haris.backendspringboottechiteasymodel.models.Television;
+import nl.haris.backendspringboottechiteasymodel.exceptions.NameTooLongException;
+import nl.haris.backendspringboottechiteasymodel.exceptions.RecordNotFoundException;
+import nl.haris.backendspringboottechiteasymodel.repositories.TelevisionRepository;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.*;
 
-import java.util.ArrayList;
 import java.util.List;
+import java.util.Optional;
 
 @RestController
 @RequestMapping("/televisions")
 public class TelevisionController {
 
-    private List<Television> televisionDataBase = new ArrayList<>();
-    private Long currentId = 1L;
+
+    private TelevisionRepository televisionRepository;
+
+    public TelevisionController(TelevisionRepository televisionRepository) {
+        this.televisionRepository = televisionRepository;
+    }
 
     // CREATE: Voeg een nieuwe television toe
     @PostMapping
-    public ResponseEntity<Television> createTelevision(@RequestBody Television televison) {
-        televison.setId(currentId++);
-        if (televison.getBrand().length() > 20) {
+    public ResponseEntity<Television> createTelevision(@RequestBody Television television) {
+
+        if (television.getBrand().length() > 20) {
             throw new NameTooLongException("the name is too big, should be less than 20 characters");
         }
 
-        televisionDataBase.add(televison);
-        return ResponseEntity.status(HttpStatus.CREATED).body(televison);
+        televisionRepository.save(television);
```

Comment:
De save methode returned een television, die kun je opslaan in een variabele als `Television savedTelevision =  televisionRepository.save(television);`